test/archaeologistApi/ArchaeologistApi.test.ts

```diff
@@ -0,0 +1,36 @@
+import { ethers } from 'ethers';
+// import { ArchaeologistApi } from '../../src/ArchaeologistApi';
+import { Libp2p } from 'libp2p';
+
+
+const signer = ethers.Wallet.createRandom({});
+
+describe('ArchaeologistApi', () => {
+  describe('ArchaeologistApi Constructor', () => {
+    test('stub', () => {});
+    // test('should initialize with a contract address, signer, and url string', () => {
```

Comment:
A `Cannot find module 'it-pipe' from 'src/ArchaeologistApi.ts'` error is prevent me from running these tests + one other failure in the `SarcoClient` test since it imports `ArchaeologistApi`